lib/commands/command.bash

```diff
@@ -9,7 +9,14 @@ fi
 
 # Load direnv stdlib if not already loaded
 if [ -z "$(declare -f -F watch_file)" ]; then
-  eval "$(asdf exec direnv stdlib)"
+  # try using direnv from PATH if already available, since
+  # this is much faster than going thru asdf.
+  # Otherwise for non-system direnv we take the slow path.
+  if command -v direnv >&2>/dev/null; then
```

Comment:
There seems to be a bug in shfmt. In the meantime, we can get around it with this?

```suggestion
  if [ -x "$(command -v direnv)" ]; then
```

Reply:
@mvdan This is probably a bug, right?
https://github.com/asdf-community/asdf-direnv/runs/5723609384?check_suite_focus=true

Reply:
Thanks for the ping; I looked at this as I was working through my notifications backlog. Should be fixed in https://github.com/mvdan/sh/pull/879; the formatter will add a space between the redirections, which will also work around the parser bug in the latest stable version.